### PR TITLE
fix(chat): fix my application version search (Issue #2648)

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- fix filter on my applications tab while searching for particular version of installed app
- fix suggested results detailed view

Issues:

- Issue #2648 
- Issue #2474 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2650.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Marketplace/TabRenderer.tsx [hunk 1]
```diff
@@ -265,31 +265,30 @@ export const TabRenderer = ({ screenState }: TabRendererProps) => {
...
-          )
+        ? filteredEntities.filter(isInstalledModel)
         : filteredEntities;
 
     const shouldSuggest =
       selectedTab === MarketplaceTabs.MY_APPLICATIONS && isSomeFilterNotEmpty;
 
     const groupedEntities = groupModelsAndSaveOrder(
-      shouldSuggest ? filteredEntities : entitiesForTab,
+      orderBy(entitiesForTab, 'version', 'desc').concat(
```
Comment (IlyaBondar) [inline apps/chat/src/components/Marketplace/TabRenderer.tsx]: why do we sort versions?

File: apps/chat/src/components/Marketplace/TabRenderer.tsx [hunk 2]
```diff
@@ -265,31 +265,30 @@ export const TabRenderer = ({ screenState }: TabRendererProps) => {
...
-    );
+    let orderedEntities = groupedEntities.map(({ entities }) => entities[0]);
 
     if (shouldSuggest) {
       const suggestedListWithoutInstalled = orderedEntities.filter(
-        (entity) => !installedModelIds.has(entity.reference),
-      );
-      orderedEntities = orderedEntities.filter((entity) =>
-        installedModelIds.has(entity.reference),
+        (entity) => !isInstalledModel(entity),
```
Comment (IlyaBondar) [inline apps/chat/src/components/Marketplace/TabRenderer.tsx:295]: image

**Comment by Gimir** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/540184)

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2650.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2650.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2650.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**